src/utils/constants.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * Keywords used in schemas that will be skipped over when 
+ * creating lines for NormalLeftRow and NormalRightRow.
+ * These are typically skipped over since they are already 
+ * illustrated in other parts of the SchemaTable
+   (ex. symbols in the left panel)
+ */
+const skipKeywords = [
+  '$id',
+  '$schema',
+  'type',
+  'name',
+  'description',
+  'items',
+  'contains',
+  'properties',
+  'required',
+  'allOf',
+  'anyOf',
+  'oneOf',
+  'not',
+];
+
+export default skipKeywords;

```

Comment:
To make sure we can add additional constants to this file, we could instead do:

```jsx
export const SKIP_KEYWORDS = [
  '$id',
  '$schema',
  'type',
  'name',
  'description',
  'items',
  'contains',
  'properties',
  'required',
  'allOf',
  'anyOf',
  'oneOf',
  'not',
];

// export const <some-other-variable>
// ...
```

You will probably have to include a eslint disable rule in this file since the linter will try to convince you to add the `export default` keyword when we're only exporting one object.

Reply:
Thanks! Applied changes!